conbench/api/_comparator.py

```diff
@@ -140,9 +151,17 @@ def __init__(self, pairs, threshold=None):
     def formatted(self):
         for pair in self.pairs.values():
             baseline, contender = pair.get("baseline"), pair.get("contender")
-            yield BenchmarkComparator(baseline, contender, self.threshold).formatted()
+            yield BenchmarkComparator(
+                baseline,
+                contender,
+                self.threshold,
```

Comment:
If you remove the trailing comma after `self.threshold,` there won't be a diff here.